index.qmd

```diff
@@ -24,42 +24,50 @@ format:
 
-In this paper we focus on the competencies of RSEs and the data scientists.
+While there is overlap between all these fields, we can summarise that RSEs focus on the creation of software in order to facilitate research using their Software Engineering (SE) skills.
+Research Data Managers focus on the data that is generated and/or used during the research process and provide infrastructure for better provisioning of the data.
+Data Scientists focus on gaining insight from the data that is generated somewhere.
+Research Infrastructure Engineers focus on provisioning and running the computing platforms that software and data experts often use to support their work.
+
+In this paper we focus on the competencies of RSEs and Data Scientists and compare and contrast the skills and activities of these two groups.
 (WHY?!)
-Similar to Research Software Engineering (RSE), Data Science (DS) is a cross-cutting field that focuses on a special area within the sciences or data oriented businesses.
 
-The major computing organizations ACM and IEEE published a joint recommended computing curriculum which already included data science: the Computing Curriculum 2020 [@CC2020] already lists DS as one of the special cases for computing competences.
+We focus our comparison in this paper on these two groups for three reasons:
+
+ - Within the digital technical professionals space, RSEs and Data Scientists are the two groups whose skills are most software focused. In this context, there can be significant crossovers in the skills developed and used, even though the focus of the two roles can be quite different.
+ - Similar to RSE, Data Science (DS) is a cross-cutting field that is applicable across a wide range of research domains. We believe it is important to understand how the application of competencies attributed to these roles may differ across domains.
+ - Despite the crossover between DS and RSE roles, we observe that the communities representing these roles are quite disconnected or independent, perhaps becuase of the different communities through which these roles have developed - this is something we are keen to understand in more detail.
```

Comment:
```suggestion
 - Despite the crossover between DS and RSE roles, we observe that the communities representing these roles are quite disconnected or independent, 
 perhaps because of the different communities through which these roles have developed - this is something we are keen to understand in more detail.
```